x11-packages/shared-mime-info/postrm

```diff
@@ -2,7 +2,7 @@
 set -e
 PREFIX="@TERMUX_PREFIX@"
 
-if [ "${1}" = purge ]; then
+if [ "${1}" = purge ] || [ "@TERMUX_PACKAGE_FORMAT@" = "pacman" ]; then
```

Comment:
```suggestion
if [[ "${1}" = purge || "@TERMUX_PACKAGE_FORMAT@" = "pacman" ]]; then
```

Reply:
`sh` is used here, not `bash`. So it is not possible to use `[[`.

Reply:
Emmm...
![image](https://github.com/user-attachments/assets/aa61a783-103f-4305-b576-cf06e80f5473)

Reply:
I apologize, I mixed up the file